src/conn/conn_oligarch.c

```diff
@@ -1032,23 +1032,20 @@ __wt_disagg_get_meta(
 {
     WT_CONNECTION_IMPL *conn;
     WT_DISAGGREGATED_STORAGE *disagg;
-    WT_ITEM result;
     WT_PAGE_LOG_GET_ARGS get_args;
     u_int count;
 
     conn = S2C(session);
     disagg = &conn->disaggregated_storage;
-    WT_CLEAR(result);
     WT_CLEAR(get_args);
 
     if (disagg->page_log_meta != NULL) {
         WT_ASSERT(session, disagg->bstorage_meta == NULL);
         count = 1;
         WT_RET(disagg->page_log_meta->plh_get(disagg->page_log_meta, &session->iface, page_id,
-          checkpoint_id, &get_args, &result, &count));
+          checkpoint_id, &get_args, item, &count));
```

Comment:
@ddanderson This change needs a little attention. I'm essentially undoing part of a previous change you made in LABS-1102, but that was a big diff and I don't understand why we needed `result` originally.

Reply:
I don't think we need it.  That's a nice simplification.